taskcluster/fenix_taskgraph/routes.py

```diff
@@ -9,9 +9,9 @@
 from taskgraph.transforms.task import index_builder
 
 SIGNING_ROUTE_TEMPLATES = [
-    "index.project.{trust-domain}.{project}.v3.{variant}.{build_date}.revision.{head_rev}",
-    "index.project.{trust-domain}.{project}.v3.{variant}.{build_date}.latest",
-    "index.project.{trust-domain}.{project}.v3.{variant}.latest",
+    "index.project.{trust-domain}.{project}.v2.{variant}.{build_date}.revision.{head_rev}",
```

Comment:
I made sure the index aren't changed by running tests locally. One exception: staging indexes aren't there anymore. I'm okay to remove them for 2 reasons:

* Managing `*.staging.*` has been difficult to me, because production scopes like `fenix.v3.*` allow staging indexes to be overwritten.
* I think taskgraph is mature enough to be relied on, in terms of indexes.

I'm okay to revisit this, if you feel otherwise @mitchhentges

Reply:
Sure I'm ok with removing staging indexes.
Is it still be possible to test automation changes before we land them in the main repository?

Reply:
Double yes! 

Staging runs still work and just don't create indexes anymore/

Testing the routes is a matter of running `taskgraph` locally with the right parameters file. This way you're ensuring that the production run still work. I'll document this thoroughly before you start migrating fftv.